SimPEG/EM/TDEM/ProblemTDEM.py

```diff
@@ -650,10 +647,170 @@ class Problem3D_e(BaseTDEMProblem):
     _formulation = 'EB'
     fieldsPair = Fields3D_e  #: A Fields3D_e
     surveyPair = SurveyTDEM
+    Adcinv = None
 
     def __init__(self, mesh, **kwargs):
         BaseTDEMProblem.__init__(self, mesh, **kwargs)
 
+    def Jtvec(self, m, v, f=None):
```

Comment:
this needs to be abstracted back to the base class, but can be addressed in a separate pr